draft-ietf-httpbis-digest-headers.md

```diff
@@ -648,69 +606,50 @@ validation of the algorithm types, number of validations, or the size of content
 
 # IANA Considerations
 
-## Establish the HTTP Digest Algorithm Values Registry {#iana-digest-algorithm-registry}
+## Establish the HTTP Structured Digest Algorithm Values Registry {#iana-digest-algorithm-registry}
 
-This memo sets this specification to be the establishing document for the [HTTP Digest
-Algorithm
-Values](https://www.iana.org/assignments/http-dig-alg/) registry.
+This memo sets this specification to be the establishing document for the [HTTP Structured Digest
```

Comment:
Should we also formally close the old registry?

Reply:
Thanks, you've highlighted an issue with section I'd missed. 

Other options tried to update the old registry. That's not the case here, I'm proposing a new registry and I need to update the text that does that to read better.

I don't know what the formalities for closing a registry are in relation to an obsoleted header. Does that mean it vanishes from IANA (seems bad), or that it can no longer have new entries added to it, or something else?

Reply:
@mnot any hint for this?

Reply:
Paging @mnot again to test his email filters

Reply:
Ah, it's `mention@`, not `mentions` -- try again?

Reply:
@mnot please answer the question 😉

Reply:
It works!

Oh, the *question*. Right. I don't know that anything needs to happen to the registry, unless you want to close it down (i.e., not allow updates). If that's the case, you should say so in IANA considerations. Generally a lot of this gets negotiated tweaked later in the process when IANA does their review.